WordPress-Extra/ruleset.xml

```diff
@@ -2,6 +2,22 @@
 <ruleset name="WordPress Extra">
 	<description>Best practices beyond core WordPress Coding Standards</description>
 
+	<rule ref="Generic.PHP.DeprecatedFunctions"/>
+	<rule ref="Generic.PHP.ForbiddenFunctions"/>
+	<rule ref="Generic.Functions.CallTimePassByReference"/>
+	<rule ref="Generic.Formatting.DisallowMultipleStatements"/>
+	<rule ref="Generic.Formatting.MultipleStatementAlignment"/>
```

Comment:
In this case, I'd optionally put `$redirect_vars_pairs` above line 129, and align the `=` out.

In other cases, just adding a clear line above 134 would do - that the separates the predefined data in `$redirect_vars`, and the variable that's only populated from the `foreach`.

It is [part of the standard](https://make.wordpress.org/core/handbook/coding-standards/php/#indentation).

However, I'm open to removing it if you both still insist.